src/tools/sss_override.c

```diff
@@ -333,13 +333,21 @@ static struct sysdb_attrs *build_attrs(TALLOC_CTX *mem_ctx,
         if (ret != EOK) {
             goto done;
         }
+    }else {
+        fprintf(stderr, ("Setting uid=0 is not allowed\n"));
+        ret = !EOK;
```

Comment:
Setting the error code to !EOK is not exactly what we expect is this case.
For instance, we want to have the error code giving as much info as possible for whoever is receiving it.

In this specific case, we're returning an error because an invalid option (uid = 0) ... so, I do believe that receiving an error as EINVAL would be more appropriate.